marlowe-playground-client/src/Blockly/Blockly.ts

```diff
@@ -0,0 +1,23 @@
+import { BlockBase, BlockCreate, BlockChange } from "blockly/core/events/events";
```

Comment:
This is a helper file for the Blockly types, as we are importing Blockly dynamically. We should revisit this approach